Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -51,8 +51,8 @@
 
 upgradeProperties[2017] = {
     '2017' : {
-        'Geom' : 'DB:Extended',
-        'GT' : 'auto:phase1_2017_realistic',
+        'Geom' : 'Extended2017dev',
+        'GT' : 'auto:phase1_2017_hcaldev',
```

Comment:
Note that #15674 changed `Extended2017` but not the dev version, so merging might be nontrivial.